### DISCUSSION TITLE
Remix is not getting installed with "npx create-remix@latest"

### DISCUSSION THREAD

**Discussion by ethiqs** [remix-run/remix#4370] [Q&A]
root@system:/var/www/html/remix# npx create-remix@latest
Need to install the following packages:
  create-remix@1.7.2
Ok to proceed? (y) y
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated rollup-plugin-inject@3.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.

after that it is exited .

**Comment by machour** [comment]
Hi @ethiqs, is this still a problem? What node version are you using?

Also, I would strongly recommend avoiding installing npm stuff using the `root` user ⚠️
